tests/test_download.py

```diff
@@ -174,10 +174,68 @@ def test__find_container_images_config_nextflow(self, tmp_path, mock_fetch_wf_co
         assert "nfcore/methylseq:1.4" in download_obj.containers
         assert "nfcore/sarek:dev" in download_obj.containers
         assert "https://depot.galaxyproject.org/singularity/r-shinyngs:1.7.1--r42hdfd78af_1" in download_obj.containers
-        # does not yet pick up nfcore/sarekvep:dev.${params.genome}, because successfully detecting "nfcore/sarek:dev"
-        # breaks the loop already. However, this loop-breaking is needed to stop iterating over DSL2 syntax if a
-        # direct download link has been found. Unless we employ a better deduplication, support for this kind of
-        # free-style if-else switches will sadly remain insufficient.
+        # does not yet pick up nfcore/sarekvep:dev.${params.genome}, because that is no valid URL or Docker URI.
+
+    #
+    # Test for 'find_container_images' in modules
+    #
+    @with_temporary_folder
+    @mock.patch("nf_core.utils.fetch_wf_config")
+    def test_find_container_images_modules(self, tmp_path, mock_fetch_wf_config):
```

Comment:
This test will be very useful, thanks!

Reply:
You're welcome! 

It will benefit me as well, as I won't be surprised by a sudden new module syntax. Once we settle on a new template syntax and modify the `modules create` and `lint` accordingly, we can also directly include one in this test to ensure that downloads can also resolve the container paths accurately.